<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interferogram_generation</class>
 <widget class="QWidget" name="interferogram_generation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>干涉相位生成</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QtWidgetsApplication3.qrc">
    <normaloff>:/QtWidgetsApplication3/bin/icon/InSARapp.ico</normaloff>:/QtWidgetsApplication3/bin/icon/InSARapp.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择工程</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_src_project"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,6">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>选择节点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_src_node"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="4,6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>选择主图像</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_master_image"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,6">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>目标节点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_dst_node"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,2,3,2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>距离向多视倍数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_multilook_rg"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>方位向多视倍数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_multilook_az"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="7,3">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../QtWidgetsApplication3.qrc"/>
 </resources>
 <connections/>
</ui>
